package.json

```diff
@@ -46,5 +46,8 @@
     "wasmdump": "./lib/bin/wasmdump.js",
     "wasmast": "./lib/bin/wasmast.js",
     "wasmrun": "./lib/bin/wasmrun.js"
+  },
+  "dependencies": {
+    "webassembly-floating-point-hex-parser": "^0.1.2"
```

Comment:
FYI https://github.com/xtuc/js-webassembly-interpreter/commit/575e72db61b36678ab6dde0433518d54b64ffc1d I had to remove the "no dep" in the README and could we fix the version?